### ISSUE TITLE
Server crash under Windows after a number of pool restarts

### ISSUE THREAD

**Issue by AlexPravdin** [spiral/roadrunner#84]
Windows 10 x64
RoadRunner 1.3
PHP 7.3.0 
Pipes relay

I've set up File Watchers in PHPStorm to restart RR's pool on each source modification with the `http:reset` command. And after some time RR always crashes in this way:

```
INFO[5926] ::1 {145ms} 500 GET http://localhost:8090/v1/render
INFO[5926] ::1 {4.80ms} 500 GET http://localhost:8090/favicon.ico
INFO[5927] ::1 {3.08ms} 500 GET http://localhost:8090/v1/render
INFO[5927] ::1 {1.99ms} 500 GET http://localhost:8090/favicon.ico
INFO[5934] ::1 {3.90ms} 500 GET http://localhost:8090/v1/render
INFO[5934] ::1 {1.55ms} 500 GET http://localhost:8090/favicon.ico
INFO[5935] ::1 {6.91ms} 500 GET http://localhost:8090/v1/render
INFO[5935] ::1 {2.39ms} 500 GET http://localhost:8090/favicon.ico
DEBU[6397] new worker pool
DEBU[6397] new worker pool
panic: close of closed channel

goroutine 118 [running]:
github.com/spiral/roadrunner.(*StaticPool).Destroy(0xc0002c40b0)
        /mnt/c/Users/Wolfy-J/go/src/github.com/spiral/roadrunner/static_pool.go:155 +0x61
github.com/spiral/roadrunner.(*Server).Reconfigure.func1(0xc00013c9b0, 0xa51fa0, 0xc0002c40b0)
        /mnt/c/Users/Wolfy-J/go/src/github.com/spiral/roadrunner/server.go:145 +0x6c
created by github.com/spiral/roadrunner.(*Server).Reconfigure
        /mnt/c/Users/Wolfy-J/go/src/github.com/spiral/roadrunner/server.go:143 +0x2ec
```

**Comment by AlexPravdin** [comment]
Thanks!

**Comment by wolfy-j** [comment]
Fixed in 1.3.1
